### PR TITLE
BETA: Register nycrp.is-a.dev

### PR DESCRIPTION
**Pull request by savageemote1**
Added `nycrp.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/nycrp.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "savageemote1",
+        "email": "jpreciado3546@gmail.com"
+    },
+    "record": {
+        "URL": "nycrp.is-a.dev/"
```
[the remark this answers is not in the document]
Comment (savageemote1) [reply domains/nycrp.json:7]: what's that exactly, 



> You can't redirect to your own domain `URL` should contain protocol

[the remark this answers is not in the document]
Comment (Stef-00012) [reply domains/nycrp.json:7]: > what's that exactly,
> 
> > You can't redirect to your own domain `URL` should contain protocol

`https://`

Comment (Stef-00012) [inline domains/nycrp.json:7]: you can't redirect the site to itself

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/nycrp.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Stef-00012) [reply domains/nycrp.json:7]: > what's that exactly,
> 
> > You can't redirect to your own domain `URL` should contain protocol

`https://`

**Comment by orxngc** [comment]
Inactive

**Pull request closed without merging**